<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecEditorDialog</class>
 <widget class="QDialog" name="DecEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dec Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>PIN_LIST</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Site Numbers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxSiteNums">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetSiteNums">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddPin">
            <property name="text">
             <string>Add Row</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelPin">
            <property name="text">
             <string>Delete Row</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="PinListTable" name="tablePinList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>TIME_NAME_DEF</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnAddTimeNameDef">
            <property name="text">
             <string>Add Row</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelTimeNameDef">
            <property name="text">
             <string>Delete Row</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableTimeNameDef">
          <column>
           <property name="text">
            <string>TimeSet Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TimeSet Interval</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DecEditorGroupWidget" name="widgetPinGroup" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="DecEditorGroupWidget" name="widgetPowerPinGroup" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="DecEditorGroupWidget" name="widgetURPinGroup" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DecEditorGroupWidget</class>
   <extends>QWidget</extends>
   <header>ui/DecEditorGroupWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PinListTable</class>
   <extends>QTableWidget</extends>
   <header>ui/PinListTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
